MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1188,6 +1183,84 @@ public string GetVideoQualityParam(EncodingJobInfo state, string videoEncoder, E
                     qmin,
                     qmax);
             }
+            else if (string.Equals(videoEncoder, "libvpx-vp9", StringComparison.OrdinalIgnoreCase)) // vp9
+            {
+                // When `-deadline` is set to `good` or `best`, `-cpu-used` ranges from 0-5.
+                // When `-deadline` is set to `realtime`, `-cpu-used` ranges from 0-15.
+                // Resources:
+                //   * https://trac.ffmpeg.org/wiki/Encode/VP9
+                //   * https://superuser.com/questions/1586934
+                //   * https://developers.google.com/media/vp9
+                switch (encodingOptions.EncoderPreset)
+                {
+                    case "veryslow":
+                        param += " -deadline best -cpu-used 0";
+                        break;
+
+                    case "slower":
+                        param += " -deadline best -cpu-used 2";
+                        break;
+
+                    case "slow":
+                        param += " -deadline best -cpu-used 3";
+                        break;
+
+                    case "medium":
+                        param += " -deadline good -cpu-used 0";
+                        break;
+
+                    case "fast":
+                        param += " -deadline good -cpu-used 1";
+                        break;
+
+                    case "faster":
+                        param += " -deadline good -cpu-used 2";
+                        break;
+
+                    case "veryfast":
+                        param += " -deadline good -cpu-used 3";
+                        break;
+
+                    case "superfast":
+                        param += " -deadline good -cpu-used 4";
+                        break;
+
+                    case "ultrafast":
+                        param += " -deadline good -cpu-used 5";
+                        break;
+
+                    default:
+                        param += " -deadline good -cpu-used 1";
+                        break;
+                }
```

Comment:
Just my personal preference. More compact.
```suggestion
                param += encodingOptions.EncoderPreset switch
                {
                    "veryslow" => " -deadline best -cpu-used 0",
                    "slower" => " -deadline best -cpu-used 2",
                    "slow" => " -deadline best -cpu-used 3",
                    "medium" => " -deadline good -cpu-used 0",
                    "fast" => " -deadline good -cpu-used 1",
                    "faster" => " -deadline good -cpu-used 2",
                    "veryfast" => " -deadline good -cpu-used 3",
                    "superfast" => " -deadline good -cpu-used 4",
                    "ultrafast" => " -deadline good -cpu-used 5",
                    _ => " -deadline good -cpu-used 1"
                };
```